### PR TITLE
Add @jhowardmsft to maintainers

### PR DESCRIPTION
**Pull request by icecrime**
How convenient is it that we can finally all react ":tada:" on pull requests in a single click, right?

Ping @jhowardmsft :-)

### CODE REVIEW

**Comment by icecrime** [comment]
Of course, merging if `windowsTP4` job is green :trollface:

File: MAINTAINERS [hunk 1]
```diff
@@ -34,6 +34,7 @@
 			"duglin",
 			"estesp",
 			"icecrime",
+			"jhoward",
```
Comment (thaJeztah) [inline MAINTAINERS]: should we use `jhowardmsft` for consistency?

**Comment by thaJeztah** [comment]
One nit/question, but LGTM. Congrats @jhowardmsft

**Comment by jhowardmsft** [comment]
Wasn't expecting to come into the office and see that this morning :dog: :smile: :uk:

File: MAINTAINERS (hunk 1 above)
Comment (calavera) [inline MAINTAINERS]: it should

**Comment by ahmetalpbalkan** [comment]
@jhowardmsft congrats!

**Pull request merged**